torchgeo/datasets/utils.py

```diff
@@ -17,14 +17,15 @@
 from collections.abc import Iterable, Iterator, Sequence
 from dataclasses import dataclass
 from datetime import datetime, timedelta
-from typing import Any, cast, overload
+from typing import Any, Union, cast, overload
 
 import numpy as np
 import rasterio
 import torch
 from torch import Tensor
 from torchvision.datasets.utils import check_integrity, download_url
 from torchvision.utils import draw_segmentation_masks
+from typing_extensions import TypeAlias
```

Comment:
We don't have a dep on typing_extensions right now. Let's just remove the `TypeAlias` usage, we don't actually need it.